src/d3a/d3a_core/redis_communication.py

```diff
@@ -39,7 +42,7 @@ def __init__(self, simulation, simulation_id):
                                    self._simulation_id + "/pause": self._pause_callback,
                                    self._simulation_id + "/resume": self._resume_callback,
                                    self._simulation_id + "/slowdown": self._slowdown_callback}
-        self.result_channel = "d3a-results"
+        self.result_channel = RESULTS_CHANNEL
```

Comment:
Sorry, I didn't understand the advantage of this?

Reply:
I wanted to try to isolate the output channels of d3a, in order for maybe in the future to move them to ConstSettings or something similar. Furthermore, it is in general good practice to never have hardcoded values anywhere in the code, even though we do that it is generally good to avoid that.